web/source/text/processor.ts

```diff
@@ -370,7 +378,7 @@ namespace com.keyman.text {
       // Safari, IE, Opera?
       //}
 
-      if(fromOSK) {
+      if(fromOSK && !keyman.isEmbedded) {
```

Comment:
Just wanted to check that `fromOSK` in ll 365 and 370 don't need to check ` && !keyman.isEmbedded`

Reply:
Line 365 should handle modifier keypresses (K_SHIFT, for example), even for external keyboards when `isEmbedded == true`.

Line 370 is performing a browser check... it shouldn't affect anything, though I guess it wouldn't hurt to disable that block.